easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('libpng', '1.6.32'),
+    ('freetype', '2.8'),
```

Comment:
We're using `freetype` 2.8.1 in existing `matplotlib` 2.1.0 easyconfigs, better keep that in sync here?